src/lib/entities/podcast/index.ts

```diff
@@ -1,2 +1,4 @@
 export { default as PodcastTile } from './ui/podcast-tile.svelte';
 export { default as PodcastDisplay } from './ui/podcast-display.svelte';
+export { transformPodcastRequest } from './api/adapters';
```

Comment:
Well, this function is not used anywhere outside of the `podcasts` entity. But the `transformEpisodeRequest` is exposed from the `episodes` entity. So I was thinking, should we expose only the things that are used from outside, or sometimes expose "just in case" for the sake of consistency?